src/AGTHARN/FlyPE/commands/FlyCommand.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/* 
+ *  ______ _  __     _______  ______ 
+ * |  ____| | \ \   / /  __ \|  ____|
+ * | |__  | |  \ \_/ /| |__) | |__   
+ * |  __| | |   \   / |  ___/|  __|  
+ * | |    | |____| |  | |    | |____ 
+ * |_|    |______|_|  |_|    |______|
+ *
+ * FlyPE, is an advanced fly plugin for PMMP.
+ * Copyright (C) 2020 AGTHARN
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace AGTHARN\FlyPE\commands;
+
+use pocketmine\command\CommandSender;
+use pocketmine\command\PluginCommand;
+use pocketmine\utils\TextFormat as C;
+use pocketmine\Player;
+
+use AGTHARN\FlyPE\Main;
+
+class FlyCommand extends PluginCommand {
```

Comment:
Be careful using `PluginCommand` for future references. It is `final` in PM4, luckily this plugin only has one command, so it won't be difficult to transfer when PM4 is out. Just be careful in the future using it with PM3.

Reply:
Okay. I will take note of it.